Containers/base-py/Dockerfile

```diff
@@ -2,24 +2,48 @@
 ARG  BASE_IMAGE=nvidia/cuda:9.2-runtime-ubuntu16.04
 FROM $BASE_IMAGE
 
-COPY ./base-py/requirements /ai4e_api_tools/requirements
-
 ENV PATH /usr/local/envs/ai4e_py_api/bin:$PATH
 ENV PYTHONPATH="${PYTHONPATH}:/ai4e_api_tools"
 
-# Install Miniconda, Flask, Supervisor, uwsgi
-RUN ./ai4e_api_tools/requirements/install-api-hosting-reqs.sh
+RUN export LANG=C.UTF-8
+RUN export LC_ALL=C.UTF-8
+
+RUN apt-get update --fix-missing \
+    && apt-get install -y software-properties-common \
+    && add-apt-repository ppa:deadsnakes/ppa \
+    && apt-get update \
+    && apt-get install -y apt-utils \
+        supervisor \
+        curl \
+        bzip2 \
+        ca-certificates \
+        libglib2.0-0 \
+        libxext6 \
+        libsm6 \
+        libxrender1 \
+        python3.7 \
+        python3-setuptools \
+        python3-dev \
+        apt-transport-https \
+        build-essential \
+        gcc 
+
+RUN easy_install3 pip
+
+# Install hosting requirements
+RUN pip3 install flask \
+        flask-restful \
+        gunicorn
 
 # Install Azure Blob SDK
-RUN ./ai4e_api_tools/requirements/install-azure-blob.sh
+RUN pip3 install azure-mgmt-storage \
```

Comment:
Should we pin the versions of the Azure packages or record them somewhere? Each time the base images are built, these versions may change.